uvtools/dspec.py

```diff
@@ -775,3 +868,71 @@ def delay_filter_leastsq(data, flags, sigma, nmax, add_noise=False,
         mdl_array[i, :] = bf_model
 
     return mdl_array, cn_array, inp_data
+
+
+def sinc_downweight_mat_inv(nchan, df, filter_centers, filter_widths,
+                            filter_factors, cache = {}, wrap = False, wrap_interval=1,
+                            nwraps = 1000, no_regularization = False):
+    """
+    Computes inverse of clean weights for a baseline.
+    This form of weighting is diagonal in delay-space and down-weights tophat regions
+
+    Parameters
+    ----------
+    nchan: integer
+        Number of channels on baseline
+    df: float
+        channel width (Hz)
+    filter_centers: float or list
+        float or list of floats of centers of delay filter windows in nanosec
+    filter_widths: float or list
+        float or list of floats of widths of delay filter windows in nanosec
+    filter_factors: float or list
+        float or list of floats of filtering factors.
+    cache: dictionary, optional dictionary storing filter matrices with keys
+    (nchan, df, ) + (filter_centers) + (filter_widths) + \
+    (filter_factors)
+
+    !!!-------------
+    WARNING: The following parameters are intended for studying weighting
+    but shouldn't not be changed from defaults in practical data analysis!
```

Comment:
typo: shouldn't not be

Reply:
Thanks for catching this. I've fixed it.